tasks/main.yml

```diff
@@ -6,6 +6,10 @@
   include: pkg_url.yml
   when: threatstack_pkg_url is undefined
 
+- name: Disable auditd service
+  include: disable_auditd.yml
+  when: ansible_distribution == 'Amazon'
```

Comment:
@Shplorf this should only be applied to Amazon Linux 2.

Reply:
I realized my mistake and the correct fix is on: https://github.com/threatstack/threatstack-ansible/pull/48/files#r255199810